dashboard/src/App.css

```
@@ -0,0 +1,963 @@
+.detail-label {
+  font-size: 11px; font-weight: 600; color: var(--text2);
+  text-transform: uppercase; letter-spacing: 0.5px;
+}
+.detail-value { font-size: 14px; }
+.detail-reason { font-size: 13px; line-height: 1.6; white-space: pre-wrap; }
+.detail-pre {
+  font-size: 13px; font-family: monospace;
+  background: var(--surface); backdrop-filter: blur(8px);
+  -webkit-backdrop-filter: blur(8px);
+  padding: 8px 12px; border-radius: 8px; margin-top: 4px;
+  white-space: pre-wrap; word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)